salt/grains/nvme.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+'''
+Grains for NVMe Qualified Names (NQN).
+
+.. versionadded:: Flourine
+
+To enable these grains set `nvme_grains: True`.
+
+.. code-block:: yaml
+
+    nvme_grains: True
+'''
+# Import Python libs
+from __future__ import absolute_import, print_function, unicode_literals
+
+import errno
+import logging
+
+# Import Salt libs
+import salt.modules.cmdmod
```

Comment:
I can't see where in this module we are running shell commands. Do we actually need this import?

Reply:
Correct. I'll remove it.